frontend/src/pages/admin/Users/Invitations.vue

```diff
@@ -12,54 +12,55 @@
 
 <script>
 import adminApi from '@/api/admin'
-import { markRaw } from "vue"
+import teamApi from '@/api/team'
+import { markRaw } from 'vue'
 import FormHeading from '@/components/FormHeading'
 import ItemTable from '@/components/tables/ItemTable'
 import InviteUserCell from '@/components/tables/cells/InviteUserCell'
 import { TrashIcon } from '@heroicons/vue/outline'
-import { useRoute, useRouter } from 'vue-router';
+import { useRoute, useRouter } from 'vue-router'
 
 const MemberInviteRemoveButton = {
-    template: `<button type="button" class="forge-button-inline px-2 py-2" @click="removeInvite"><TrashIcon class="w-5" /></button>`,
-    props: ['id','team','onremove'],
+    template: '<button type="button" class="forge-button-inline px-2 py-2" @click="removeInvite"><TrashIcon class="w-5" /></button>',
+    props: ['id', 'team', 'onremove'],
     components: {
         TrashIcon
     },
     methods: {
-        async removeInvite() {
+        async removeInvite () {
             await this.onremove(this.team.id, this.id)
         }
     }
 }
 
 export default {
     name: 'UserInviteTable',
-    data() {
+    data () {
         return {
             invitations: [],
             inviteColumns: [
-                {name: 'User', component: { is: markRaw(InviteUserCell) }, property: 'invitee'},
-                {name: 'Team', property:'teamName'},
-                {name: 'Invited By',property:'invitor', component: { is: markRaw(InviteUserCell) }},
-                {name: 'Expires In', property: 'expires'},
-                {name: '', class: ['w-16'], component: {is: markRaw(MemberInviteRemoveButton) } }
+                { name: 'User', component: { is: markRaw(InviteUserCell) }, property: 'invitee' },
+                { name: 'Team', property: 'teamName' },
+                { name: 'Invited By', property: 'invitor', component: { is: markRaw(InviteUserCell) } },
+                { name: 'Expires In', property: 'expires' },
+                { name: '', class: ['w-16'], component: { is: markRaw(MemberInviteRemoveButton) } }
             ]
         }
     },
-    mounted() {
+    mounted () {
         this.fetchData()
     },
     methods: {
-        async removeInvite(teamId, inviteId) {
-            // await adminApi.removeTeamInvitation(teamId, inviteId)
-            // await this.fetchData();
+        async removeInvite (teamId, inviteId) {
+            await teamApi.removeTeamInvitation(teamId, inviteId)
+            await this.fetchData()
```

Comment:
With all the linting changes mixed in here it is quite hard to see the material changes.

I think it is just these two lines uncommented?

Reply:
Yes, sorry, have linting turned on with saving of the file. The changes are:
- uncommenting the `await this.fetchData()`
- using the `teamApi` instead of the `adminApi`

Reply:
Looks good.

I propose a task is scheduled for Friday that applies the lint rules to all frontend files and a PR raised to get them applied. We need to move in one step rather than bit by bit.